src/main/java/com/wolox/training/controllers/UserController.java

```diff
@@ -0,0 +1,67 @@
+package com.wolox.training.controllers;
+
+import com.wolox.training.controllers.vo.UsersVO;
+import com.wolox.training.models.Users;
+import com.wolox.training.services.UserService;
+import java.util.List;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.stereotype.Controller;
+import org.springframework.web.bind.annotation.DeleteMapping;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.PutMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.bind.annotation.ResponseBody;
+import org.springframework.web.bind.annotation.ResponseStatus;
+
+@Controller
+@RequestMapping("/api/v1/users")
+public class UserController {
+
+    @Autowired
+    private UserService userService;
+
+    @PostMapping
+    public @ResponseBody
+    ResponseEntity<Users> create(@RequestBody UsersVO usersVO) {
+        return new ResponseEntity<>(userService.create(usersVO), HttpStatus.CREATED);
+    }
+
+    @PostMapping("/{id}/addBook")
```

Comment:
I think that in this case, and in the removeBook, the method could be an Patch or a Put maybe.

We can reach an agreement, but for Post method, you have to use a RequestBody

Reply:
Thanks Mati! It slipped my mind! :)